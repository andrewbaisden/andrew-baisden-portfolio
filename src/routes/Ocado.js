import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import PortfolioPageOcado from '../components/PortfolioPageOcado';
import CheckPortfolioPage from '../components/CheckPortfolioPage';
// import PortfolioCarouselOcado from '../components/PortfolioCarouselOcado';

const Ocado = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="An agency approached me with a job, working for Ocado in Hatfield. In this role, I was working alongside another developer, and a web designer."
        />
        <title>Ocado - Portfolio - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/portfolio/ocado" />
      </Helmet>
    </div>
    <CheckPortfolioPage />
    <Header />
    <PortfolioPageOcado companyBG="#ffffff" />
    {/* <PortfolioCarouselOcado /> */}
    <Footer />
  </div>
);

export default Ocado;
