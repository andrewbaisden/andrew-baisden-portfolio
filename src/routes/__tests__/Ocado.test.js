import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Ocado from '../Ocado';

Enzyme.configure({ adapter: new Adapter() });

describe('<Ocado />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Ocado />);
    expect(wrapper).toMatchSnapshot();
  });
});
