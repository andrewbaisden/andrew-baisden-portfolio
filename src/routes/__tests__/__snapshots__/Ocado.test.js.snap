// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ocado /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
